esadapter/src/main/java/com/ncc/neon/adapters/es/ElasticsearchQueryConverter.java

```diff
@@ -426,6 +429,20 @@ public static UpdateRequest convertMutationByIdQuery(MutateQuery mutateQuery) {
             .doc(mutateQuery.getFieldsWithValues());
     }
 
+    public static UpdateByQueryRequest convertMutationByFilterQuery(MutateQuery mutateQuery) {
+        SelectClause selectClause = new SelectClause(mutateQuery.getDatabaseName(), mutateQuery.getTableName());
+        QueryBuilder queryBuilder = convertWhereClauses(selectClause, List.of(mutateQuery.getWhereClause()));
+        UpdateByQueryRequest request = new UpdateByQueryRequest();
+        request.setQuery(queryBuilder);
+        request.getSearchRequest().indices("_all");
```

Comment:
Same comment I made in the NEON-65 PR about using `mutateQuery.getTableName()` instead of `_all` here.

Reply:
Fixed.